common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -669,7 +669,8 @@ def convert(content, input_format, output_format):
 
             if output_format == 'txt':
                 text = json.loads(content)['text']
-                return HTMLParser().unescape("\n".join(text))
+                text_without_none = [line if line else '' for line in text]
```

Comment:
Again, what about this?
`'\n'.join(filter(None, text))`

Reply:
@awaisdar001 we don't want to skip the line. Just want it to be blank